src/index.js

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './calculator.css';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] According to project requirements,  Once the Calculator is ready, you should import it and use it in your top-level component: **App.js**  in React case, the main component, the one that represents your entire application/web-site. “App.js” would usually refer to just that, your main component. Please fix it 👍